wooey/tasks.py

```diff
@@ -234,7 +234,11 @@ def cleanup_dead_jobs():
 
     # Get active tasks from Celery
     inspect = celery_app.control.inspect()
-    active_tasks = {task['id'] for worker, tasks in six.iteritems(inspect.active()) for task in tasks}
+    task_infos = inspect.active()
+    if task_infos == None:
```

Comment:
It might be worth having this line as `if task_infos is None:` according to https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or